### PR TITLE
Update customer-support.md

### PR DESCRIPTION
**Pull request by craigwillis-mm**
Updates to the Support Team pages.  Images to be added later!

### CODE REVIEW

**Review by emilyhollinger — DISMISSED** [review]

File: operations/business-operations/customer-support.md [hunk 1]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
 
-To ensure that all issues raised with the Mattermost Support Team can be easily tracked, updated, and resolved the Support Team uses [Zendesk](https://www.zendesk.com/) to manage cases.
-Customers can access the Support Portal and submit/view their tickets via https://support.mattermost.com. 
+To ensure that all issues raised with the Mattermost Support Team can be easily tracked, updated, and resolved, the Support Team uses Zendesk as their case management software.
 
-**New Users to Mattermost Support**
+Customers can access the Support Portal and submit/view their tickets via https://support.mattermost.com.  
 
-If you are new to Mattermost Support you can [click here](https://mattermost.zendesk.com/auth/v2/login/registration?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc) to set up and verify your account.
+## New Users to Mattermost Support ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## New Mattermost Support users
```

File: operations/business-operations/customer-support.md [hunk 2]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
 
-**Existing Users of Mattermost Support**
+If you are new to Mattermost Support you can click [here](https://mattermost.zendesk.com/auth/v2/login/registration?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc) to set up and verify your account.
 
-If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can [click here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
+## Existing Users of Mattermost Support ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Existing Mattermost Support users
```

File: operations/business-operations/customer-support.md [hunk 3]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can click [here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used. To (re)set your password, you can click [here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
```

File: operations/business-operations/customer-support.md [hunk 4]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+## Ticket Management ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Ticket management 
```

File: operations/business-operations/customer-support.md [hunk 5]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+## Ticket Creation ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
### Ticket creation 
```

File: operations/business-operations/customer-support.md [hunk 6]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+When submitting a Support Ticket to the Mattermost Support Team we recommend that you use the ‘Submit a Request’ option from with the Support Portal.  This will ensure that we get the required information from you.
+
+## Support Ticket Request Form ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
### Support ticket request form 
```

File: operations/business-operations/customer-support.md [hunk 7]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
-Details of what the Support Team provides for our Enterprise Customers can be found here, this includes our support options and SLAs.
+Details of what the Support Team provides for our Enterprise Customers can be found [here](https://mattermost.com/support/), this includes our support options and SLA’s.
 
-**Meet the Team**
+## Meet the Team: 
 
 ![](../../.gitbook/assets/Support-team-image-for-handbook.png)
 
-**Where to find us**
+## Where to find us:
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Where to find us
```

File: operations/business-operations/customer-support.md [hunk 8]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
 
 The team is based across three continents to ensure a follow the sun approach to working with our customers.  They also have strong relationships with all parts of the Mattermost organization to ensure that where necessary issues are addressed quickly and efficiently, these include, but are not limited to, Customer Success, Engineering, and Product Management.
 
-Details of what the Support Team provides for our Enterprise Customers can be found here, this includes our support options and SLAs.
+Details of what the Support Team provides for our Enterprise Customers can be found [here](https://mattermost.com/support/), this includes our support options and SLA’s.
 
-**Meet the Team**
+## Meet the Team:
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Meet the team
```

File: operations/business-operations/customer-support.md [hunk 9]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
-**Where to find us**
+## Where to find us:
 
 * Mattermost Channels: Customer Support, Support Standup
-* Support Team Email: support@mattermost.com
+* Enterprise Support Team Email: support@mattermost.com
 * Customer Support Portal: https://support.mattermost.com
 
-**Mattermost Ticketing System**
+## Mattermost Ticketing System
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Mattermost ticketing system 
```

File: operations/business-operations/customer-support.md [hunk 10]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
-If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can [click here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
+## Existing Users of Mattermost Support ##
+
+If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can click [here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
+
+## Ticket Management ##
+
+## Ticket Creation ##
+
+When submitting a Support Ticket to the Mattermost Support Team we recommend that you use the ‘Submit a Request’ option from with the Support Portal.  This will ensure that we get the required information from you.
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
When you submit a Support Ticket to the Mattermost Support Team, you can use **Submit a Request** in the Support Portal. This ensures that we get the information we need to assist you.
```

File: operations/business-operations/customer-support.md [hunk 11]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+## Support Ticket Request Form ##
+
+To ensure that we can assist our customers most efficiently it’s imperative that as much information is provided when submitting the ticket.  For details of what details should be provided you should review our Troubleshooting Guide which can be found [here](https://docs.mattermost.com/guides/administrator.html#troubleshooting).
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
To ensure that we can assist our customers most efficiently it's important that as much information as possible is provided when submitting the ticket. More details around the required information is listed in our [Troubleshooting Guide](https://docs.mattermost.com/guides/administrator.html#troubleshooting).
```

File: operations/business-operations/customer-support.md [hunk 12]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+|Field | Description |
+|------|-------------|
+| **CC**  |Add additional email addresses of those who you would like to receive notifications of this issue.|
+| **Subject** |A brief description of the issue.            |
+| **Priority**     |Urgent, High, Normal, or Low, descriptions of these priority ratings can be found here.              |
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Priority**     |Urgent, High, Normal, or Low, descriptions of these priority ratings can be found here.              |
```
Will there be a link provided for the descriptions? Or will we link to a section of this document?

File: operations/business-operations/customer-support.md [hunk 13]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+| **Category**     |Select a category that best fits the product area affected by the issue raised.            |
+| **Description**     |Provide as much detail as possible to enable the Support Team to recreate the issue.  Please refer to the troubleshooting guide for what’s required.            |
+| **Environment**     |Select the environment where you are experiencing the issue.            |
+| **Mattermost Version**     |Enter the number of the Mattermost server version you are currently running.            |
+| **Mattermost Mobile version**     |If the issue you are reporting is on a mobile device, enter the version here.            |
+| **Mobile device**     |Please select which type of device you are experiencing the issue on.            |
+| **Attachments**     |Add screenshots, logs, etc to the ticket here.            |
+
+ ## Ticket Workflow ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
 ## Ticket workflow
```

File: operations/business-operations/customer-support.md [hunk 14]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+While working with our customers the Support Team will us the following ticket statuses to manage the ticket, New, Open, Pending, Solved, and Closed.  The workflow will be as follows:
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
The Support Team use the following ticket statuses to manage customer tickets: New, Open, Pending, Solved, and Closed. The workflow will be as follows:
```
- Is the workflow going to be added or are we referring to the steps/content that follows this sentence? 
- Are the ticket statuses listed referring to UI elements in Zendesk (asking with regard to formatting the words if needed)?

File: operations/business-operations/customer-support.md [hunk 15]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+Note: Descriptions of each status can be found [here](https://handbook.mattermost.com/operations/business-operations/analytics/metrics-definitions#support-tickets).
+ 
+## Issue Escalation Process ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md:62]:
```suggestion
## Issue escalation process
```

File: operations/business-operations/customer-support.md [hunk 16]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+Some technical issues may need to be addressed by our Engineering Team.  When this occurs you will be notified via the Support Ticket and the following internal steps will be carried out.
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
Some technical issues may need to be addressed by our Engineering Team for further investigation. If this is the case, you will be notified via the Support Ticket and the following internal steps will be carried out.
```
Are the "internal steps" referring to the content below or will steps be added?

File: operations/business-operations/customer-support.md [hunk 17]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+## Customer Support/Sustaining Engineering Team ##
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
## Customer Support/Sustained Engineering Team
```

File: operations/business-operations/customer-support.md [hunk 18]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+Any case that can’t be resolved by the Support Team is escalated to the Mattermost Sustaining Engineering Team for their review.
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
Any case that can’t be resolved by the Support Team is escalated to the Mattermost Sustained Engineering Team (SET) for their review.
```

File: operations/business-operations/customer-support.md [hunk 19]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+**If the issue is confirmed as a product issue then a Pull Request is created and where necessary members of the following Teams are involved:**
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
**If the issue is confirmed as a product issue then a Pull Request is created and, where necessary, members of the following teams are involved:**
```

File: operations/business-operations/customer-support.md [hunk 20]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+- Support
+- Sustaining Engineering
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
- Sustained Engineering
```

File: operations/business-operations/customer-support.md [hunk 21]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+ - Product
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
- Product
```

File: operations/business-operations/customer-support.md [hunk 22]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+ - Customer Success
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
- Customer Success
```

File: operations/business-operations/customer-support.md [hunk 23]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+ - Executives
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
- MLTx
```

File: operations/business-operations/customer-support.md [hunk 24]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+**Action Plan Created**
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
**Action plan created**
```

File: operations/business-operations/customer-support.md [hunk 25]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+ - The customer is updated via the Support Ticket with the plan to address the issue.
+
+**Action Plan Executed**
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
**Action plan executed**
```

File: operations/business-operations/customer-support.md [hunk 26]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+ - The fix is tested internally
+ - An update is provided via the Support Ticket
+
+**The fix is Moved to Production**
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
**The fix is moved to Production**
```

File: operations/business-operations/customer-support.md [hunk 27]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+ - The fix is released to the latest Production version of Mattermost for download 
+ - An update is provided via the Support Ticket
+
+**Urgent & High Ticket Escalation Process for Premier Support Customers**
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
**Urgent and high priority ticket escalation process for Premium Support customers**
```

**Review by justinegeffen — COMMENTED** [review]
This is a great document with really helpful information!

I've made some general formatting suggestions and also added some questions around the content and references to workflows. Happy to discuss further.

File: operations/business-operations/customer-support.md (hunk 12 above)
Comment (craigwillis-mm) [reply operations/business-operations/customer-support.md]: Yes, it'll be updated to point to here: 

https://mattermost.com/support/

File: operations/business-operations/customer-support.md (hunk 14 above)
Comment (craigwillis-mm) [reply operations/business-operations/customer-support.md]: 1. Yes, I'm working with Jason to upload the images now.
2. Yes

File: operations/business-operations/customer-support.md (hunk 16 above)
Comment (craigwillis-mm) [reply operations/business-operations/customer-support.md]: They refer to sub-steps of the steps below.

File: operations/business-operations/customer-support.md [hunk 28]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+### Support ticket request form 
+
+To ensure that we can assist our customers most efficiently it's important that as much information as possible is provided when submitting the ticket. More details around the required information is listed in our [Troubleshooting Guide](https://docs.mattermost.com/guides/administrator.html#troubleshooting).
+
+|Field | Description |
+|------|-------------|
+| **CC**  |Add additional email addresses of those who you would like to receive notifications of this issue.|
+| **Subject** |A brief description of the issue.            |
+| **Priority**     |Urgent, High, Normal, or Low, descriptions of these priority ratings can be found here.              |
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Priority**     |Urgent, High, Normal, or Low, descriptions of these priority ratings can be found [here](https://mattermost.com/support/).              |
```

File: operations/business-operations/customer-support.md [hunk 29]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
 
 The team is based across three continents to ensure a follow the sun approach to working with our customers.  They also have strong relationships with all parts of the Mattermost organization to ensure that where necessary issues are addressed quickly and efficiently, these include, but are not limited to, Customer Success, Engineering, and Product Management.
 
-Details of what the Support Team provides for our Enterprise Customers can be found here, this includes our support options and SLAs.
+Details of what the Support Team provides for our Enterprise Customers can be found [here](https://mattermost.com/support/), this includes our support options and SLA’s.
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
Details of what the Support Team provides for our Enterprise Customers can be found [here](https://mattermost.com/support/), this includes our support options and SLAs.
```

File: operations/business-operations/customer-support.md [hunk 30]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
 
 The team is based across three continents to ensure a follow the sun approach to working with our customers.  They also have strong relationships with all parts of the Mattermost organization to ensure that where necessary issues are addressed quickly and efficiently, these include, but are not limited to, Customer Success, Engineering, and Product Management.
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
The team is based across three continents to ensure a follow-the-sun approach to working with our customers. They have strong relationships with all parts of the Mattermost organization including, but not limited to, Customer Success, Engineering, and Product Management. These partnerships ensure that issues are addressed quickly and efficiently. 
```

File: operations/business-operations/customer-support.md [hunk 31]
```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
...
+
+Any case that can’t be resolved by the Support Team is escalated to the Mattermost Sustained Engineering Team (SET) for their review.
+
+**If the issue is confirmed as a product issue then a Pull Request is created and, where necessary, members of the following teams are involved:**
+
+- Support
+- Sustained Engineering
+- Product
+- Customer Success
+- MLTx
```
Comment (justinegeffen) [inline operations/business-operations/customer-support.md]:
```suggestion
- Mattermost Leadership Team (MLT)
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @craigwillis-mm, this looks good to me. Let me know if you're ready to merge this iteration. :)

**Comment by craigwillis-mm** [comment]
> Thanks, @craigwillis-mm, this looks good to me. Let me know if you're ready to merge this iteration. :)

Yes, this can be merged.

Craig

**Pull request merged**